.travis/codecoverage.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+# fail script immediately on any errors in external commands
+set -e
+
+source travis_retry.sh
```

Comment:
where does that script come from?

Reply:
my understanding is that it should be available based on [this](https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/templates/header.sh).

Reply:
nvm, going to have to [copy the function](https://twitter.com/travisci/status/499196774712901632)

Reply:
do we even need `travis_retry` here? it makes sense to use it for the initial `npm install` step, but if coverage reporting  deps fail it's not sooo bad, right?